internal/webhook/v1alpha/conduit_webhook.go

```diff
@@ -149,10 +153,16 @@ func (*ConduitCustomDefaulter) proccessorDefaulter(pp []*v1alpha.ConduitProcesso
 
 // ConduitCustomValidator struct is responsible for validating the Conduit resource
 // when it is created, updated, or deleted.
-type ConduitCustomValidator struct{}
+type ConduitCustomValidator struct {
+	validationService validator.Validator
```

Comment:
You can likely just embed this into the structure e.g. 
```suggestion
	validator.Validator
```

Reply:
Updated